PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/api/rendering/PrebidDisplayView.java

```diff
@@ -127,7 +136,7 @@ public void onClickThroughClosed(
     };
     public PrebidDisplayView(
             @NonNull Context context,
-            DisplayViewListener listener,
+            @Nullable DisplayViewListener listener,
```

Comment:
the `@Nullable` doesn't correspond interface declaration. Does it make sense to change the interface?